packages/common/refresh-runtime.js

```diff
@@ -545,6 +555,13 @@ function isLikelyComponentType(type) {
   }
 }
 
+function isPlainObject(obj) {
```

Comment:
Is there a reason we need this?
`typeof obj === "object" && obj !== null` should be enough no?

Reply:
Maybe, but in your condition, values such as Date, [], Map, etc will return true as well, which is not what we want. Or maybe it's an oversight on my end?

Reply:
Hum true, it will still be possible to loop over keys but it's cleaner to not do it for these so we can keep this